app/helpers/application_helper.rb

```diff
@@ -2,4 +2,8 @@ module ApplicationHelper
   def render_condition(condition)
     render partial: 'condition', locals: {condition: condition}
   end
+
+  def render_params(params, empty_string)
```

Comment:
The name of this confused me, since it's not rendering the `params` itself. Would `render_config` or `render_hash` work for you?

Reply:
yeah for sure, that's easy to change